server/views/probationPractitionerReferrals/referralCancellationReason.njk

```diff
@@ -10,6 +11,10 @@
   - GOV.UK{% endblock %}
 
 {% block content %}
+   {% if serviceUserNotificationBannerArgs !== undefined and serviceUserNotificationBannerArgs !== null %}
```

Comment:
do we have this check elsewhere?

Reply:
I took it from `actionPlanFormTemplate.njk` where the check is probably more useful as it's a partial that we extend in a few places so it might be undefined/null in those places, whereas here it'll be defined. I'm tempted to leave it as-is though as I've put it in all parts of this journey, until we extract it out into a more re-usable component, then we can replace it everywhere.